cmd/build/v2/services.go

```diff
@@ -27,45 +27,43 @@ import (
 )
 
 // GetServicesToBuild returns the services it has to built because they are not already built
-func (bc *OktetoBuilder) GetServicesToBuild(ctx context.Context, manifest *model.Manifest, svcToDeploy []string) ([]string, error) {
+func (bc *OktetoBuilder) GetServicesToBuild(ctx context.Context, manifest *model.Manifest, servicesToDeploy []string) ([]string, error) {
 	buildManifest := manifest.Build
 
 	// check if images are at registry (global or dev) and set envs or send to build
-	toBuild := make(chan string, len(buildManifest))
+	nonBuiltServices := make(chan string, len(buildManifest))
 	g, _ := errgroup.WithContext(ctx)
 	for service := range buildManifest {
 
 		svc := service
 		g.Go(func() error {
-			return bc.checkServicesToBuild(svc, manifest, toBuild)
+			return bc.checkServicesToBuild(svc, manifest, nonBuiltServices)
 		})
 	}
 
 	if err := g.Wait(); err != nil {
 		return nil, err
 	}
-	close(toBuild)
+	close(nonBuiltServices)
 
-	if len(toBuild) == 0 {
+	if len(nonBuiltServices) == 0 {
```

Comment:
I think this name is misleading. I understand `nonBuiltServices` as services that weren't built, but if this list is empty, we say that images were already built 🤯 

previous name (`toBuild`) means images that have to be build, so if that list is empty, we display the message, which is correct because we don't have any image to build

Reply:
You are right. I also think that `toBuild` is misleading, since we do some handling after it and those values change. I'll check if I come up with some better names for this. My objective is to make it easier to read next time we work on it

Reply:
I'm having a hard time thinking of something better. The name is logically correct, and for the message we got: if there aren't any services unbuilt, we know that all of them were built. I know that the double negation can be confusing, but it makes sense 

I'm open to suggestions

Reply:
This is a bit of personal matter. For me, `toBuild` is clearer because it contains the list of images to be built. If it is empty, it means that the images were already built, if it has some images, it means that there are some images to build.

In this specific condition, if nonBuiltServices (services that weren't built) is empty, I understand that I have to build all the services, but it is the opposite, I don't have to do it, because they were already built.

`imagesToBuild` could be a name. At the end, this calculates the list of images that have to be built from the build section and from the services section

Reply:
IMO `toBuild` fits with the content inside it. Maybe there is some handling after checking the value but in the end the information stored in it is the same but with another format (channel->set->slice)

Reply:
> IMO `toBuild` fits with the content inside it. Maybe there is some handling after checking the value but in the end the information stored in it is the same but with another format (channel->set->slice)

`imagesToBuild` even gives more context. I see it better

Reply:
The thing is that `toBuild` or `imagesToBuild` should refer to what the function will return. The function `checkServicesToBuild` does not actually return the "images to build", since we'll take into account the "services to deploy" and do different logic

Reply:
I've changed this and I tried to follow the naming as close as it was before. I want to focus more on the logic around this to reduce the probability of introducing bugs.